cli.js

```diff
@@ -3,10 +3,8 @@
 const { Command, Argument } = require('commander');
 const program = new Command();
 const supportedLanguages = require('./SUPPORTED-LANGUAGE.json');
-const { translate } = require('./utils/translate');
 const { configPath } = require('./utils/getConfigPath');
-const { getOrCreateTranslationFile } = require('./utils/getTranslationFile');
-const fs = require('fs');
+const { translateController } = require('./controllers/translateController');
```

Comment:
I think moving these should have been another PR before, but no problem keeping the change

Reply:
Yea I agree that would be better on another PR. When I was creating the tests for this file I found with the problem than mocking functions for CLI commands didn't work 😅, so I had to separate this logic of the CLI interface to be able to test it individually

Reply:
I really appreciate you helping me with the review. Thanks!